src/run_monitor.js

```diff
@@ -1,14 +1,13 @@
 #!/usr/bin/env node
-
 const death = require('death');
-const Monitor = require('../dist/monitor.js').default;
+const Monitor = require('../dist/src/monitor.js').default;
```

Comment:
Why is this path changing?

Reply:
Tsc creates a path in dist like this, so dist/src/filename.js

Maybe some aditional configuration is needed here so that it "flattens" the path?

Reply:
Oh if it works, great. I'm just confused about how it resolved before without this...

Reply:
I'm not sure to be honest, it just stopped working... :/